docs/rubocop.md

```diff
@@ -40,12 +40,14 @@ Consider an example:
 it { is_expected.to be_success }
 it { is_expected.to have_header('X-TOTAL-PAGES', 10) }
 it { is_expected.to have_header('X-NEXT-PAGE', 2) }
+its(:status) { is_expected.to eq(200) }
```

Comment:
Let's add a note to the doc about `rspec-its` with the link to the gem. People might be confused that this is a built-in RSpec feature